src/Dapr.Actors/Runtime/ActorRuntime.cs

```diff
@@ -65,6 +68,44 @@ public void RegisterActor<TActor>(Func<ActorTypeInformation, ActorService> actor
             this.actorManagers[actorTypeInfo.ActorTypeName] = new ActorManager(actorService);
         }
 
+        /// <summary>
+        /// 
+        /// </summary>
+        public void UseActorSettings(ActorSettings settings)
+        {
+            ArgumentVerifier.ThrowIfNull(settings, nameof(settings));
+
+            this.actorSettings = new ActorSettings(
+                settings.ActorIdleTimeout,
+                settings.ActorScanInterval,
+                settings.DrainOngoingCallTimeout,
+                settings.DrainRebalancedActors);
+        }
+
+        internal Task SerializeSettingsAndRegisteredTypes(System.Buffers.IBufferWriter<byte> output)
+        {
+            using Utf8JsonWriter writer = new Utf8JsonWriter(output);
+            writer.WriteStartObject();
+
+            writer.WritePropertyName("entities");
+            writer.WriteStartArray();
+
+            foreach (var actorType in this.RegisteredActorTypes)
+            {
+                writer.WriteStringValue(actorType);
+            }
+
+            writer.WriteEndArray();
+
+            if (this.actorSettings != null)
```

Comment:
Could you please add a comment that actor settings is only set when provided by user using the UseActorSettings method, would have to look at the code to understand this.